### ISSUE TITLE
Customizer: add `title` attributes to icon-only controls

### ISSUE THREAD

**Issue by rickybanister** [Automattic/wp-calypso#46693]
There are a couple of usability/accessibility concerns with icon-only controls in the customizer:

<img width="467" alt="Screen Shot on 2020-10-22 at 08-42-50" src="https://user-images.githubusercontent.com/437258/96910870-166afb00-146e-11eb-996b-774ee880e14c.png">

The AMP toggle here could use a title attribute or tooltip to explain that it turns on the AMP preview mode. 

The responsive viewport icons are a common pattern, but they could also benefit from a title or tooltip. 

Thanks to @rralian for the initial report.

**Comment by obenland** [comment]
Unless we can find someone from outside of Manage who'd be interested in advancing this upstream, I'm afraid this probably doesn't have a realistic chance to be fixed any time soon. From what I understand both parts come from upstream repositories where this should be addressed and there are just too many other tickets of higher priority before Manage would be able to start looking into this.

**Comment by worldomonation** [comment]
Given the comment above, I would propose closing this issue with label `Closed - Won't Fix`.
